src/ui/components/viewControl.vue

```diff
@@ -46,6 +47,18 @@ export default {
     controlClass: {
       type: String,
       default: 'c-disclosure-triangle'
+    },
+    domainObject: {
+      type: Object,
+      default: () => {}
+    }
+  },
+  computed: {
+    ariaLabelValue() {
+      const name = this.domainObject.name ? ` ${this.domainObject.name}` : '';
+      const type = this.domainObject.type ? ` ${this.domainObject.type}` : '';
+
+      return `${this.value ? 'Collapse' : 'Expand'}${name}${type}`;
```

Comment:
super nitpick and doesn't really matter, but we should put the spaces down here instead of in `name` and `type`

Reply:
Oh I actually did that on purpose, because tree isn't the only thing that uses this, so this way it would just say "Expand" or "Collapse" if no domain object is passed in.

Reply:
SHIP IT! 🚀

Reply:
... but run the linter first